### PR TITLE
bundle analysis: calculate bundle's total gzip size correctly

### PR DESCRIPTION
**Pull request by JerrySentry**
Accurately compute a bundle's gzip size.

### Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. In 2022 this entity acquired Codecov and as result Sentry is going to need some rights from me in order to utilize my contributions in this PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

### CODE REVIEW

File: requirements.in [hunk 1]
```diff
@@ -20,7 +20,7 @@ factory-boy
 fakeredis
 freezegun
 https://github.com/codecov/opentelem-python/archive/refs/tags/v0.0.4a1.tar.gz#egg=codecovopentelem
-https://github.com/codecov/shared/archive/42f83ec717e632c543cc6ceab8fc3cc39eccc5be.tar.gz#egg=shared
+https://github.com/codecov/shared/archive/87495b40c8e0af14a9d093946aa3cac2094b1de8.tar.gz#egg=shared
```
Comment (JerrySentry) [inline requirements.in:23]: to be updated once corresponding shared PR is merged

File: services/bundle_analysis.py [hunk 2]
```diff
@@ -263,6 +263,10 @@ def asset(self, name: str) -> Optional[AssetReport]:
     def size_total(self) -> int:
         return self.report.total_size(**self.filters)
 
+    @cached_property
+    def gzip_size_total(self) -> int:
```
Comment (ajay-sentry) [inline services/bundle_analysis.py:267]: jw, is this int or float?

**Review by ajay-sentry — APPROVED** [review]
looks good

File: services/bundle_analysis.py (hunk 2 above)
Comment (JerrySentry) [reply services/bundle_analysis.py:267]: int, it represents number of bytes

Comment (ajay-sentry) [reply services/bundle_analysis.py:267]: got it 👌 thanks for the context

**Pull request merged**